WeakAuras/AuraEnvironment.lua

```diff
@@ -173,6 +173,9 @@ local blockedFunctions = {
 
 local blockedTables = {
   SlashCmdList = true,
+  SendMailMailButton = true,
+  SendMailMoneyGold = true,
```

Comment:
That sounds like a function

Reply:
![image](https://user-images.githubusercontent.com/189656/103351597-05f58300-4aa4-11eb-89a9-e8505f91bceb.png)